### DISCUSSION TITLE
Is possible add schema to Data API using the CLI and avoid the UI of Supabase?

### DISCUSSION THREAD

**Discussion by Andres6936** [supabase/supabase#29163] [Questions] [answered]
Is possible add a custom schema to Data API using the CLI and avoid the UI of Supabase?

<img width="1038" alt="image" src="https://github.com/user-attachments/assets/d48611b9-03cd-4d48-851e-ff8cb82e09d4">

I prefer that my script of bootloader added the customs schemas that will be creating to Data API.

**Comment by GaryAustin1** [comment, marked as answer]
https://supabase.com/docs/guides/cli/config
and
The management API https://supabase.com/docs/reference/api/v1-update-postgrest-service-config

**Reply by Andres6936** [reply]
Thaaannnks!!!!
